scripts/workspace_status.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+"""Bazel `--workspace_status_command` script.
+
+This script is designed to be used in Bazel`s `--workspace_status_command` and
+generate any desirable status artifacts.
+"""
+
+__copyright__ = """
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""
+
+import subprocess
+
+
+def git_commit_sha() -> str:
+    return subprocess.check_output(
+        ["git", "rev-parse", "--short", "HEAD"], shell=False, text=True
```

Comment:
```suggestion
        ["git", "rev-parse", "--short", "HEAD"], encoding="utf-8"
```

`shell=False` is the default, and I think they're trying to move away from text (e.g., https://discuss.python.org/t/jep-400-utf-8-by-default-and-future-of-python/14246)